src/assets/i18n/en.json

```diff
@@ -556,61 +556,61 @@
         "BLOG_HEADING": "What's New",
         "VIEW_BLOG_LINK": "Read more on the blog"
       },
-      "COLLECTIONS": [{
-          "SUBHEADING": "Copyright in Artstor",
-          "DESCRIPTION": "Learn more with our guide",
-          "CAPTION": "Artstor LibGuides",
-          "LINK": "http://artstor.libguides.com/Copyright-in-Artstor/",
-          "IMG_SOURCE": "/assets/img/featured-collections/Copyright_LibGuide.jpg",
-          "LINK_TITLE": "Copyright in Artstor",
-          "ALT_TEXT": "Copyright in Artstor"
+      "COLLECTIONS": [
+        {
+        "SUBHEADING": "The art of medicine",
+        "DESCRIPTION": "Eakins’ medical paintings and clinical fact",
```

Comment:
nit - indents here might be a smidge off with the `{` at the same level as its contents.